### PR TITLE
feat(world): add more functionality to manage the world's entities

### PR DESCRIPTION
**Pull request by anarchuser**
Invoking the .kill(ID) method moves individuals to the cemetery

### CODE REVIEW

File: test/model/world/world.cpp [hunk 1]
```diff
@@ -6,62 +6,86 @@
...
                 }
             }
         }
     }
-}
\ No newline at end of file
+}
+
+SCENARIO("The world can freely be modified") {
+    GIVEN("And instantiated and filled world") {
```
Comment (xeruf) [inline test/model/world/world.cpp]:
```suggestion
    GIVEN("an instantiated and filled world") {
```

File: test/model/world/world.cpp [hunk 2]
```diff
@@ -6,62 +6,86 @@
...
+                    REQUIRE_THROWS_AS(world->addIndividual(new Fred(0, 0, 10)), std::overflow_error);
                 }
             }
         }
     }
-}
\ No newline at end of file
+}
+
+SCENARIO("The world can freely be modified") {
```
Comment (xeruf) [inline test/model/world/world.cpp]: I'm not quite sure that this should be a separate scenario. If you share variables between scenarios, it should probably be only one.

**Review by xeruf — CHANGES_REQUESTED** [review]

File: test/model/world/world.cpp (hunk 2 above)
Comment (anarchuser) [reply test/model/world/world.cpp]: true

**Comment by xeruf** [comment]
do you think "world" should be its own scope and not simply "model"?

**Review by simon-amadeus — APPROVED** [review]

**Review by simon-amadeus — APPROVED** [review]

**Review by simon-amadeus — APPROVED** [review]

**Review by simon-amadeus — APPROVED** [review]

**Review by xeruf — APPROVED** [review]

**Pull request merged**